code/modules/admin/IsBanned.dm

```diff
@@ -137,6 +137,11 @@ Yogs End*/
 				
 				qdel(query_get_bound_creds)
 
+				if(SSechelon.is_match(ckey, address))
+					log_access("Failed Login: [key] [computer_id] [address] - Blocked due to proxy")
+					key_cache[key] = FALSE
+					return list("reason" = "proxy", "desc" = "Your connection has been blocked due to using a VPN or proxy service. If this was done in error or if you are required to use a VPN to connect to the server, please visit [CONFIG_GET(string/banappeals) || "the forums"].")
```

Comment:
Clarify that disabling the vpn is an option

Reply:
Done